_sass/_components/_footer.scss

```diff
@@ -1,9 +1,16 @@
+$footer-max-width: 1440px;
```

Comment:
Probably you want this to be rem.
Or not? Just think about it for a sec and update if you are agree.

Ideally, the footer should keep its proportions if the user change the font size preferences. I am not totally sure but if the user set more font size and the footer has this value in px the footer may be broken?

Not sure but, check and update if need it.

Reply:
Let me try rems. The idea is the same, limit the width of the footer on bigger screens. This caused links to look odd due to grid gutter

Reply:
I changed this to use a bootstrap variable instead.

```scss
  > .container-fluid {
    max-width: map-get($container-max-widths, xl);
  }
```